generate_test_coverage.sh

```diff
@@ -118,11 +118,12 @@ if [ "$PRINT_COVERAGE" = true ]; then
     fi
     echo "Info: Detected the following Test executables: $TEST_FILES"
 
-    DOC_TEST_BINS=$(find $DIR/target/debug/ -name rust_out)
-    if ! echo $DOC_TEST_BINS | grep . -q ; then
-        echo "ERROR: doc wasn't build exiting"
-        exit -1
-    fi
+    # Checks if doctests were actually built - currently we don't have any though
```

Comment:
Why?

Reply:
Ok, I've now read the comment.
I don't know if we should rather:
- remove doctests (seriously, we won't have any in the near to middle future)
- keep the code

Reply:
If you want, I can remove this code, but I'd rather keep it. That would make this coverage config more complete.